tests/bluetooth/bsim_bt/compile.source

```diff
@@ -8,9 +8,7 @@ function compile(){
   local conf_file="${conf_file:-prj.conf}"
   local conf_overlay="${conf_overlay:-""}"
 
-  local cmake_args="${cmake_args:-"-DCONFIG_COVERAGE=y \
-    -DCONFIG_DEBUG_OPTIMIZATIONS=y \
-    -DCMAKE_EXPORT_COMPILE_COMMANDS=ON"}"
```

Comment:
Please leave in the CMAKE_EXPORT_COMPILE_COMMANDS, it's a useful feature when using clang based tools to analyze the source.